### ISSUE TITLE
data-dir option for MacOS

### ISSUE THREAD

**Issue by duhanov** [monero-project/monero#9206]
Please, activate data-dir option in Gui-Wallet for MacOS 🙏

**Comment by selsta** [comment]
You can set the blockchain directory in settings which in turn sets the --data-dir flag.

**Comment by duhanov** [comment]
It would be great if it were possible to specify via the console like this:
/Applications/monero-wallet-gui.app/Contents/MacOS/monero-wallet-gui --data-dir=/Volumes/mydisk/Monero/

but now this is not possible (
Unknown option 'data-dir'.

**Comment by selsta** [comment]
Yes, `--data-dir` is a monerod flag and not a monero-wallet-gui flag. Is there a good reason why this should be added to GUI?

**Comment by duhanov** [comment]
I would not like my wallet settings and location to be visible. The ideal option is for the client itself and settings to be stored on an external drive. My solution: store the launch script on an external drive indicating the path to the wallet, for this I need --data-dir parameter

**Comment by selsta** [comment]
Did you try to use the monero-gui portable mode?

**Comment by selsta** [comment]
Put monero-gui on your external drive and start it. Click on "Change wallet mode" in the main menu, and then select the "portable mode" flag. Afterwards all settings will be stored in the same location as the monero-gui program itself.
